runtime/compiler/optimizer/VectorAPIExpansion.cpp

```diff
@@ -2874,6 +2916,40 @@ TR::Node *TR_VectorAPIExpansion::transformRORtoROL(TR_VectorAPIExpansion *opt, T
    return subNode;
    }
 
+bool
+TR_VectorAPIExpansion::getConvertSourceType(TR_VectorAPIExpansion *opt, TR::Node *node,
+                                            TR::DataType &sourceElementType, vec_sz_t &bitsLength)
+   {
+   TR::Compilation *comp = opt->comp();
+
+   // For convert, source vector type info is in children 2 and 3
+   TR::Node *sourceElementTypeNode = node->getChild(2);
+   sourceElementType = getDataTypeFromClassNode(comp, sourceElementTypeNode);
+
+   TR::Node *sourceNumLanesNode = node->getChild(3);
+
+   if (sourceNumLanesNode->getOpCode().isLoadConst())
+      {
+      int32_t elementSize = OMR::DataType::getSize(sourceElementType);
+      bitsLength = sourceNumLanesNode->get32bitIntegralValue()*8*elementSize;
+
+      if (supportedOnPlatform(comp, bitsLength) == TR::NoVectorLength)
+         {
+         traceMsg(comp, "Platform does not support conversion source length %d in node %p",
```

Comment:
This trace message is missing a trailing newline, and similarly in the other failure case just below. I know they were already like that, but it seems reasonable to fix them while moving these lines, especially considering that other parts of this commit make similarly minor fixes

Reply:
Fixed both lines.